### ISSUE TITLE
Plugin not showing up in HS-Decktracker. Please explain step 4. en detail!

### ISSUE THREAD

**Issue by FLG123** [rembound/Arena-Helper#161]
So, I was able to copy the plugin "autosquelch" manually in the right folder but it's not showing up inside HS-Decktracker. So I tried step 4...
Your explanation, unfortunately, is not complete :(

Your quote: 
> 4. If the plugin does not show up in that list, right click the ‘[PLUGINNAME].dll’, go into Properties and click “Unblock” at the bottom.

Now, where do I find [PLUGINNAME].dll !? 
- I tried in the plugin folder [windows explorer] and I couldn't find any *.dll file
- I tried and searches EVERYWHERE in the HS-Decktracker and I couldn't find any "[PLUGINNAME].dll"
Please, explain, where I can find that "[PLUGINNAME].dll" en detail and PLEASE update your explanation on the site, as I'm sure I am not the only one.

Installing a plugin is taking me longer than 30-60 min, OMG. Please help!
Thx, have a nice day

**Comment by FLG123** [comment]
ok thx, I solved it, took me only 1h.. 
I downloaded the wrong autosquelch file from the "main list" - "autosquelch-master-sth"..
Then I found another issue/thread... and I got the 2016 "Autosquelch.dll" file

If possible please, in the main-plugin list/site do only offer the Autosquelch.dll file and not the Autosquelch-master for download!
This is very confusing for any new user :((
